README.md

```diff
@@ -144,9 +144,12 @@ t2t-decoder \
   --hparams_set=$HPARAMS \
   --output_dir=$TRAIN_DIR \
   --decode_hparams="beam_size=$BEAM_SIZE,alpha=$ALPHA" \
-  --decode_from_file=$DECODE_FILE
+  --decode_from_file=$DECODE_FILE \
+  --decode_to_file=translation.en
 
-cat $DECODE_FILE.$MODEL.$HPARAMS.beam$BEAM_SIZE.alpha$ALPHA.decodes
+# Eval BLEU
+# (Always report proper BLEU in papers, not the internal approx_bleu.)
+t2t-bleu --translation=translation.en --reference=ref-translation.de
```

Comment:
so where is this ref-translation.de file? this command won't actually work for a user out of the box, right? or am i missing something?

Reply:
OK. Added `echo -e 'Hallo Welt\nAuf Wiedersehen Welt' > ref-translation.de`

Reply:
perfect, thank you